news/branch.rst

```diff
@@ -0,0 +1,25 @@
+*Added:**
+
+* <for _i in range(1, {stmt}):
+        pass>
+
+**Changed:**
+
+* <news item>
+
+**Deprecated:**
+
+* <news item>
+
+**Removed:**
+
+* <for _i in _it:
```

Comment:
the content added here will end up in the release notes. Please make it more descriptive to tell what is exactly changed/added

Reply:
Ok. Sure.